tests/collector/test_app.py

```diff
@@ -0,0 +1,98 @@
+import socket
+import time
+from multiprocessing import Process
+from typing import Optional
+
+import pandas as pd
+import pytest
+
+from evidently.collector.app import run
+from evidently.collector.client import CollectorClient
+from evidently.collector.config import CollectorConfig
+from evidently.collector.config import IntervalTrigger
+from evidently.collector.config import ReportConfig
+from evidently.test_suite import TestSuite
+from evidently.tests import TestNumberOfOutRangeValues
+
+COLLECTOR_ID = "1"
+PROJECT_ID = "2"
+
+HOST = "localhost"
+PORT = 8080
+BASE_URL = f"http://{HOST}:{PORT}"
+
+
+def create_test_data() -> pd.DataFrame:
```

Comment:
can be fixtures

Reply:
I don't use fixtures in this case because using functions is more flexible, as it allows for the use of parameters. If a slightly different type of data is needed in the future, there will be no need to create test_data_2 and all the initialization logic is in 1 function.